spec/run_cd4pe_job_spec.rb

```diff
@@ -157,75 +157,6 @@
       expect(ENV['REPO_DIR']).to eq("#{@working_dir}/cd4pe_job/repo")
     end
   end
-
```

Comment:
I moved all of the container specific tests to separate specs to allow them to be tested on runners with runtimes present.